src/OpenTelemetry.ResourceDetectors.AWS/OpenTelemetry.ResourceDetectors.AWS.csproj

```
@@ -12,17 +12,9 @@
 
   <ItemGroup>
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
-    <PackageReference Include="System.Net.Http" Version="4.3.4" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimumSupportedVersion)'">
```

What about other netframework versions such as `net47`, `net471`, etc.?

```suggestion
  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '$(.NETFramework)'">
```

---

Fixed in b70b121a72c3720f262ba5f32926664abc2c0a90.
I was not aware about `TargetFrameworkIdentifier`.